src/main/java/core/basesyntax/service/FileCsvWriter.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileCsvWriter implements Writer {
```

Comment:
```suggestion
public class FileWriterImpl implements FileWriter {
```
we can actually write any content to any file, not only csv